Description: Mention the -debug package split
 In Debian systems, the special debug libraries (with extra assertions,
 checks and logging, not suitable for production) are placed in a separate
 binary package with the "-debug" suffix. These are only meant to be
 installed when specific troubleshooting is required. Instructions for this
 troubleshooting are included in the library manpages, and we add a note
 explaining that the respective -debug package should be installed.
Author: Andreas Hasenack <andreas@canonical.com>
Forwarded: no
Last-Update: 2020-01-19

--- pmdk-1.8~rc1.orig/doc/libpmem/libpmem.7.md
+++ pmdk-1.8~rc1/doc/libpmem/libpmem.7.md
@@ -283,6 +283,11 @@ typical way to access the debug version
 controlled using the following environment variables. These variables have
 no effect on the non-debug version of the library.
 
+>NOTE:
+On Debian/Ubuntu systems, this extra debug version of the library is
+shipped in the respective **-debug** Debian package and placed in
+the **/usr/lib/$ARCH/pmdk_dbg/** directory.
+
 + **PMEM_LOG_LEVEL**
 
 The value of **PMEM_LOG_LEVEL** enables trace points in the debug version
--- pmdk-1.8~rc1.orig/doc/libpmemobj/libpmemobj.7.md
+++ pmdk-1.8~rc1/doc/libpmemobj/libpmemobj.7.md
@@ -222,6 +222,11 @@ typical way to access the debug version
 controlled using the following environment variables. These variables have
 no effect on the non-debug version of the library.
 
+>NOTE:
+On Debian/Ubuntu systems, this extra debug version of the library is
+shipped in the respective **-debug** Debian package and placed in
+the **/usr/lib/$ARCH/pmdk_dbg/** directory.
+
 + **PMEMOBJ_LOG_LEVEL**
 
 The value of **PMEMOBJ_LOG_LEVEL** enables trace points in the debug version
--- pmdk-1.8~rc1.orig/doc/libpmempool/libpmempool.7.md
+++ pmdk-1.8~rc1/doc/libpmempool/libpmempool.7.md
@@ -180,6 +180,11 @@ typical way to access the debug version
 controlled using the following environment variables. These variables have
 no effect on the non-debug version of the library.
 
+>NOTE:
+On Debian/Ubuntu systems, this extra debug version of the library is
+shipped in the respective **-debug** Debian package and placed in
+the **/usr/lib/$ARCH/pmdk_dbg/** directory.
+
 + **PMEMPOOL_LOG_LEVEL**
 
 The value of **PMEMPOOL_LOG_LEVEL** enables trace points in the debug version
